lib/chat_api_web/controllers/conversation_controller.ex

```diff
@@ -139,6 +139,7 @@ defmodule ChatApiWeb.ConversationController do
 
     with {:ok, %Conversation{} = conversation} <-
            Conversations.update_conversation(conversation, conversation_params) do
+      Helpers.send_conversation_state_update(conversation, conversation_params)
```

Comment:
can we wrap these calls in a `Task.start` so that they run async? i think i'd rather not block on these for now, and i also think it's not too big a deal if the call fails... what do you think?

Reply:
yeah a slack notification doesn't seem mission-critical so I'm fine if it fails as well.